### PR TITLE
Topic/update to kirkstone

### PR DESCRIPTION
**Pull request by mdrodrigo**
Update the shellhub to kirkstone yocto release

### CODE REVIEW

File: recipes-core/shellhub/shellhub-agent_0.9.1.bb [hunk 1]
```diff
@@ -4,15 +4,18 @@ LICENSE = "Apache-2.0"
 LIC_FILES_CHKSUM = "file://${S}/src/${GO_IMPORT}/LICENSE.md;md5=fa818a259cbed7ce8bc2a22d35a464fc"
 DEPENDS = "libxcrypt"
 
+
```
Comment (otavio) [inline recipes-core/shellhub/shellhub-agent_0.9.1.bb:7]: Please drop this new empty line.

File: recipes-core/shellhub/shellhub-agent_0.9.1.bb [hunk 2]
```diff
@@ -35,6 +35,7 @@ GO_LDFLAGS = '-ldflags="${GO_RPATH} ${GO_LINKMODE} -X main.AgentVersion=v${PV} -
 GOBUILDFLAGS:append = " -modcacherw"
 
 do_compile[dirs] += "${B}/src/${GO_IMPORT}/agent"
+do_compile[network] = "1"
```
Comment (otavio) [inline recipes-core/shellhub/shellhub-agent_0.9.1.bb:38]: Does it fails otherwise? It shouldn't.

**Review by otavio — CHANGES_REQUESTED** [review]

File: recipes-core/shellhub/shellhub-agent_0.9.1.bb (hunk 1 above)
Comment (mdrodrigo) [reply recipes-core/shellhub/shellhub-agent_0.9.1.bb]: Revised

File: recipes-core/shellhub/shellhub-agent_0.9.1.bb (hunk 2 above)
Comment (mdrodrigo) [reply recipes-core/shellhub/shellhub-agent_0.9.1.bb:37]: Yes. with the network does not enable during the compile task the shellhub doesn't compile. An error occurs to access the golang proxy.
`proxy.golang.org on 10.5.3.1:53: dial udp 10.5.3.1:53: connect: network is unreachable `

Comment (otavio) [reply recipes-core/shellhub/shellhub-agent_0.9.1.bb:37]: Ok.

**Review by otavio — APPROVED** [review]

**Pull request merged**